runtime/src/bank.rs

```diff
@@ -1257,9 +1255,6 @@ pub struct Bank {
     /// latest rent collector, knows the epoch
     rent_collector: RentCollector,
 
-    /// initialized from genesis
-    epoch_schedule: EpochSchedule,
```

Comment:
Pretty sure CI will barf on this change, removing `epoch_schedule` from`Bank` breaks snapshot backwards compatibility in particular.

Reply:
Instead we could keep the field in but rename it to `_obsolete_epoch_schedule`

Reply:
It was removed from the struct, but the serialization still has the field there. It's unused in deserialization, and we use the genesis_config.epoch_schedule to set the rent_collector.epoch_schedule as well.

So we are still writing snapshots w/ the same format, and can read snapshots with that format as well, but the initialization from the snapshot is done differently.

It's still unclear to me how the genesis_configs and rent_collectors got out of sync on devnet in the first place though.

Reply:
Ah, ok cool.  Glad I'm wrong :)

> It's still unclear to me how the genesis_configs and rent_collectors got out of sync on devnet in the first place though.

How are they different?

Reply:
The slots_per_epoch on the rent_collector were 8192 in the snapshot, but 432000 off the genesis config. Same for the leader offset.

Reply:
8192 is `DEFAULT_DEV_SLOTS_PER_EPOCH`.  Maybe it's been like that forever on devnet. Oh well, I'm pretty confident this was a devnet-only configuration issue somewhere in the past